docusaurus.config.js

```diff
@@ -144,6 +144,17 @@ const config = {
         additionalLanguages: ['docker'],
       },
     }),
+  plugins: [
+    [
+      "docusaurus-plugin-remote-content",
+      {
+        name: "examples",
+        sourceBaseUrl: "https://raw.githubusercontent.com/rancher/elemental/main/examples/quickstart/",
```

Comment:
those were just dropped :/

we had duplicated docs as the docs started in the emelental repo and then moved here, so probably we had some PRs to those older docs that were lost in transition

Reply:
Currently, the examples are different in the two repos and I expected the main repo to contain the examples to run the application.

Where should the examples live because duplication can cause problems (e.g. missing location selector)